terraform/services.tf

```diff
@@ -50,7 +50,9 @@ module "project-services" {
     "meshconfig.googleapis.com",
     "meshtelemetry.googleapis.com",
     "monitoring.googleapis.com",
+    "pubsub.googleapis.com",
     "sourcerepo.googleapis.com",
+    "spanner.googleapis.com",
```

Comment:
We should enable these services in the module because we don't always need them.

Reply:
Done